doc/api/perf_hooks.md

```diff
@@ -56,10 +59,14 @@ to mark specific significant moments in the Performance Timeline.
 ### `performance.measure(name, startMark, endMark)`
 <!-- YAML
 added: v8.5.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/32651
+    description: Make `startMark` and `endMark` parameters optional.
 -->
 
 * `name` {string}
-* `startMark` {string}
+* `startMark` {string | undefined}
 * `endMark` {string}
```

Comment:
In the case of conforming to w3c performance definition, both `startMark` and `endMark` can be omitted.